client/src/components/LoginSession/LoginSession.js

```diff
@@ -25,10 +26,12 @@ function LoginSession(props) {
         props.logout();
     }
 
-    const created = moment(props.Created);
-    const createdElapsed = moment.utc(props.Created).fromNow();
-    const lastAccessed = moment(props.LastAccessed);
-    const lastAccessedElapsed = moment.utc(props.LastAccessed).fromNow();
+    moment.locale(ss.i18n.currentLocale);
```

Comment:
That's how [`<TableView />`](https://github.com/silverstripe/silverstripe-asset-admin/blob/6751560a28d03ca1f81e0e90cf52de1773f1c616/client/src/containers/TableView/TableView.js#L388) does it in asset-admin.

I would expect this to provide better result long term.

```suggestion
    moment.locale(i18n.detectLocale());
```

Reply:
Updated